<?xml version="1.0" encoding="utf-8"?><testsuites name="pytest tests"><testsuite name="pytest" errors="0" failures="2" skipped="0" tests="34" time="0.520" timestamp="2025-07-28T12:38:20.567692+02:00" hostname="DESKTOP-3QOLC10"><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_chiller_initialization_default" time="0.004" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_chiller_initialization_custom" time="0.001" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_get_status" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_connect_success" time="0.004" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_connect_failure" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_disconnect_success" time="0.004" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_disconnect_no_connection" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_dev_success" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_dev_not_connected" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_set_param_success" time="0.003" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_set_param_failure" time="0.005" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_set_param_not_connected" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_temp" time="0.003" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_set_temp" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_pump_level" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_cooling" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_keylock" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_running" time="0.006" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_status" time="0.003" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_read_stat_diagnose" time="0.003" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_set_temperature" time="0.003" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_set_pump_level_valid" time="0.003" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_set_pump_level_invalid" time="0.005" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_set_keylock" time="0.004" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_start_device" time="0.003" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_stop_device" time="0.003" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_custom_logger" time="0.003" /><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_hk_monitor" time="0.005"><failure message="AssertionError: Calls not found.&#10;Expected: [call('hk_test', 'COM3', 'Cur_Temp', 25.5, 'degC'),&#10; call('hk_test', 'COM3', 'Set_Temp', 25.5, 'degC'),&#10; call('hk_test', 'COM3', 'Run_Stat', 'RUNNING', ''),&#10; call('hk_test', 'COM3', 'Dev_Stat', 'OK', ''),&#10; call('hk_test', 'COM3', 'Pump_Lvl', 3, ''),&#10; call('hk_test', 'COM3', 'Col_Stat', 'ON', '')]&#10;  Actual: [call('hk_test', 'COM3', 'Cur_Temp', 25.5, 'degC')]">tests\chiller\test_chiller.py:393: in test_hk_monitor
    mock_custom_logger.assert_has_calls(expected_calls)
..\..\anaconda3\envs\ESIBD\Lib\unittest\mock.py:976: in assert_has_calls
    raise AssertionError(
E   AssertionError: Calls not found.
E   Expected: [call('hk_test', 'COM3', 'Cur_Temp', 25.5, 'degC'),
E    call('hk_test', 'COM3', 'Set_Temp', 25.5, 'degC'),
E    call('hk_test', 'COM3', 'Run_Stat', 'RUNNING', ''),
E    call('hk_test', 'COM3', 'Dev_Stat', 'OK', ''),
E    call('hk_test', 'COM3', 'Pump_Lvl', 3, ''),
E    call('hk_test', 'COM3', 'Col_Stat', 'ON', '')]
E     Actual: [call('hk_test', 'COM3', 'Cur_Temp', 25.5, 'degC')]</failure></testcase><testcase classname="tests.chiller.test_chiller.TestChiller" name="test_hk_monitor_with_exceptions" time="0.005"><failure message="Failed: DID NOT RAISE &lt;class 'Exception'&gt;">tests\chiller\test_chiller.py:409: in test_hk_monitor_with_exceptions
    with pytest.raises(Exception, match="Read error"):
E   Failed: DID NOT RAISE &lt;class 'Exception'&gt;</failure></testcase><testcase classname="tests.chiller.test_chiller.TestChillerCommands" name="test_read_commands" time="0.001" /><testcase classname="tests.chiller.test_chiller.TestChillerCommands" name="test_write_commands" time="0.001" /><testcase classname="tests.chiller.test_chiller.TestChillerIntegration" name="test_full_operation_cycle" time="0.005" /><testcase classname="tests.chiller.test_chiller.TestChillerIntegration" name="test_error_handling_without_connection" time="0.002" /><testcase classname="tests.chiller.test_chiller.TestChillerIntegration" name="test_logger_creation_and_usage" time="0.003" /></testsuite></testsuites>